src/vscode-bicep/src/commands/deploy.ts

```diff
@@ -1,5 +1,6 @@
 // Copyright (c) Microsoft Corporation.
```

Comment:
This file needs breaking up.  It could happen later.

Reply:
+1

Created https://github.com/Azure/bicep/issues/7087